documentation/writing_tasks.md

```diff
@@ -439,10 +439,12 @@ error. Bolt merges the 'file' and 'line' keys with results unless 'file' is pres
 
 ### Returning sensitive data
 
-To return secrets from a task, use the `_sensitive` key in the output. Bolt
-will treat the result as sensitive and won't allow it to be printed to the
-console or log.
+To return secrets from a task, use the `_sensitive` key at the top level of the output. Bolt treats
+the result as sensitive and does not allow it to be printed to the console or log. Note that Bolt
+does not obfuscate data under `_sensitive` if `_sensitive` is not at the top level.  For example,
+Bolt does not treat this `user` data as sensitive: `{user: _sensitive: { carmen: sandiego } }`.
 
+This task will generate a random password for user 'someone', making the password hash sensitive.
```

Comment:
```suggestion
This task generates a random password for user 'someone', making the password hash sensitive.
```

Reply:
💀

Reply:
I use future tense _all the time_ too haha. It's the most common correction Heston makes to my docs changes. 😅